src/applications/using-docker-images.md

```diff
@@ -47,10 +46,10 @@ Use `deis create` to create an application on the [controller][].
 Use `deis pull` to deploy your application from [DockerHub][] or
 a public registry.
 
-    $ deis pull gabrtv/example-go:latest
+    $ deis pull <username>/example-dockerfile-http:latest
     Creating build...  done, v2
 
-    $ curl -s http://example-go.local3.deisapp.com
+    $ curl -s http://example-dockerfile-http.local3.deisapp.com
```

Comment:
The `local3.deisapp.com` is a DNS entry set up years ago for quick testing of Deis v1 actually. I think that gives us a clue as to why this doc was outdated. 👴 Still, it's fine an as example domain.